### PR TITLE
Fixed typo in README

### PR DESCRIPTION
**Pull request by vornet**

### CODE REVIEW

File: services/graphite/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The special value _measurement_ is used to define the measurement name.  It can
 
 `servers.localhost.cpu.loadavg.10`
 * Template: `.host.resource.measurement*`
-* Output:  _measurement_ =`loading.10` _tags_ =`host=localhost resource=cpu`
+* Output:  _measurement_ =`loadingavg.10` _tags_ =`host=localhost resource=cpu`
```
Comment (beckettsean) [inline services/graphite/README.md]: Good find, but shouldn't it be `loadavg`, not `loadingavg` or `loading`?

Comment (vornet) [inline services/graphite/README.md]: Oops, you're right. Thanks!

**Comment by beckettsean** [comment]
@vornet have you signed the CLA?

**Comment by vornet** [comment]
@beckettsean Didn't realized I had to.  I just signed it.  I guess this is my first commit to any open source project ever. Thanks!

**Comment by mark-rushakoff** [comment]
Thanks for contributing @vornet! Welcome to the open source community!

**Pull request merged**